src/transport.ts

```diff
@@ -127,7 +127,7 @@ export class Transport extends EventEmitter implements Transport {
         if (this.channel) {
             let channel = this.channel;
             return new Promise((resolve, reject) => {
-                const result = channel.publish(exchange, routingKey, body, {persistent: true},
+                const result = channel.publish(exchange, routingKey, body, {persistent: true, contentType: "application/vnd.masstransit+json"},
```

Comment:
Can we get a unit test on this? I know that the unit test coverage is low for this project but any help there would be appreciated.

Reply:
I've added some tests for the publish mechanism as well as the delayed publishing when sending messages before the channel connection was established.